services/src/main/java/org/exoplatform/news/NewsTargetingServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package org.exoplatform.news;
+
+import org.exoplatform.social.metadata.MetadataService;
+import org.exoplatform.social.metadata.model.Metadata;
+
+import java.util.*;
+
+/**
+ * Service managing News Targeting
+ */
+public class NewsTargetingServiceImpl implements NewsTargetingService {
+
+  public static final long        NEWS_TARGETS_TYPE           =  4;
```

Comment:
the name should used instead, see remark in the social PR https://github.com/Meeds-io/social/pull/107. To be moved to utils